crates/solvers/src/tests/baseline/bal_liquidity.rs

```diff
@@ -202,6 +202,22 @@ async fn weighted_v3plus() {
                     "gasEstimate": "88892",
                     "version": "v3Plus",
                 },
+                {
+                    "kind": "constantProduct",
+                    "tokens": {
+                        "0x9c58bacc331c9aa871afd802db6379a98e80cedb": {
+                            "balance": "1000000000000000000000",
+                        },
+                        "0xe91d153e0b41518a2ce8dd3d7944fa863463a97d": { // native token on gnosis chain
+                            "balance": "1000000000000000000000",
+                        }
+                    },
+                    "fee": "0.0025",
+                    "id": "1",
+                    "address": "0x21d4c792ea7e38e0d0819c2011a2b1cb7252bd98",
+                    "router": "0x7a250d5630b4cf539739df2c5dacb4c659f2488d",
+                    "gasEstimate": "88892",
+                },
```

Comment:
Whenever there is no reference token price (ie quote requests), we have to also provide liquidity pools for [sell_token,native_token]